test/unit/textbox.js

```diff
@@ -353,6 +353,14 @@
       'lines match splitByGrapheme charSpacing -100'
     );
   });
+  QUnit.test('Measure words', function(assert) {
+    const textbox = new fabric.Textbox('word word\nword\nword', { width: 300 });
+    const { wordsData, largestWordWidth } = textbox.measureWords(textbox.textLines);
+    assert.deepEqual(
+      wordsData[0],
+      [{ word: 'word', width: largestWordWidth }, { word: 'word', width: largestWordWidth }]
+    );
+  });
```

Comment:
started to add this test but i don't understand why the word comes out as split `['w', 'o', 'r', 'd']`

Reply:
oh i see why, to measure we need to split.